Header  header
int32   field1 #34
float32 field2 #16.32
string  field3 #"kljene"
MigratedExplicit field4 #(17, 58.2 "aldfkja", 82)
